src/js/_enqueues/admin/inline-edit-post.js

```diff
@@ -128,8 +128,17 @@ window.wp = window.wp || {};
 			inlineEditPost.edit( this );
 		});
 
+		// Clone quick edit categories for the bulk editor.
+		var qeCategories = $( '#inline-edit fieldset.inline-edit-categories' );
```

Comment:
```suggestion
		var beCategories = $( '#inline-edit fieldset.inline-edit-categories' ).clone();
```

Just a tiny suggestion to remove a single use var.